packages/react-variant-view/src/store/VariantStore.ts

```diff
@@ -33,6 +33,7 @@ export class VariantStore {
             return await this.genomeNexusClient.fetchVariantAnnotationGET({
                 variant: this.variant,
                 fields: ANNOTATION_QUERY_FIELDS,
+                isoformOverrideSource: 'mskcc',
```

Comment:
add `@observable public isoformOverrideSource: string = 'mskcc'` and refer it here as `this.isoformOverrideSource`?